build/SourceLink.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+  <PropertyGroup>
+    <SourceLinkCreate>true</SourceLinkCreate>
+    <SourceLinkNoAutoLF>true</SourceLinkNoAutoLF>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="SourceLink.Create.GitHub" Version="2.0.0-*" />
```

Comment:
@davidfowl Just to confirm, is only has stuff in the `build` folder. `PrivateAssets="all"` is needed for `build` only packages?